build_tools/travis/install.sh

```diff
@@ -53,11 +53,11 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$INSTALL_MKL" == "true" ]]; then
         conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
             numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
-            cython=$CYTHON_VERSION libgfortran mkl
+            cython=$CYTHON_VERSION libgfortran mkl flake8
```

Comment:
not sure if it's worth moving this to the if below because we only need it in the one test? Actually the two kinds of tests have pretty different requirements, right? The flake8 test doesn't even need numpy and scipy I guess? But maybe that doesn't matter so much and we should keep it simple.